.github/workflows/release.yaml

```
@@ -153,23 +152,26 @@ jobs:
           } >> $GITHUB_ENV
         fi
 
-    - name: Checkout homebrew repo
-      uses: actions/checkout@v2
-      with:
-        token: ${{ secrets.JRNL_BOT_TOKEN }}
-        repository: ${{ env.FORMULA_REPO }}
+    - name: Tap formula
+      run: |
+        brew tap ${FORMULA_REPO}
+        echo '::debug::Set tap directory'
+        echo "BREW_TAP_DIRECTORY='$(brew --repository)/Library/Taps/${FORMULA_REPO}'" >> $GITHUB_ENV
```

```suggestion
        echo "BREW_TAP_DIRECTORY='$(brew --repository ${FORMULA_REPO})'" >> $GITHUB_ENV
```

`brew --repository OWNER/TAP` or `brew --repository OWNER/homebrew-TAP ` will return the whole `$(brew --repository)/Library/Taps/OWNER/homebrew-TAP` path for you.